crates/egui/src/viewport.rs

```diff
@@ -432,6 +433,13 @@ impl ViewportBuilder {
         self
     }
 
+    /// macOS: Set to `true` to allow the window to be moved by dragging the background.
+    #[inline]
```

Comment:
https://github.com/user-attachments/assets/15c76a6e-da36-4290-a734-30fc5d69ecef

It doesn't really work well with sliders and draggable widgets, but I guess that's expected. 

But maybe add a note to the comment that it doesn't, otherwise people might expect that to work and open issues about it.